### ISSUE TITLE
Former hanging 

### ISSUE THREAD

**Issue by MorganCloud** [iann0036/former2#39]
Hello,

Thank you for this wonderful tool. I am trying to get some detail in CF about IAM and SES and also SQS, but it shows the list of queues in SQS for example but it is greyed out, I cannot do anything else.
Please assist.

I see logs I have added as an attachment

I am using chrome, I tried adding the former extension and then disabling it, and neither works...

I tried firefox and same thing, I also 2 different computers.

![f2](https://user-images.githubusercontent.com/20691667/70583420-ad4b0380-1b8b-11ea-93a8-f958e7dd343f.png)
[former2.com-1576028259710.log](https://github.com/iann0036/former2/files/3947977/former2.com-1576028259710.log)

I am guessing that apart of the below message means that I need to add get queue attribute permissions?

`Error calling SQS.getQueueAttributes. Access to the resource https://sqs.us-east-1.amazonaws.com/ is denied.
datatables.js:260 AccessDenied: Access to the resource https://sqs.us-east-1.amazonaws.com/ is denied.
`

For IAM I see the following, not sure what happened, it used to all work for IAM

[former2.com-2576028683490.log](https://github.com/iann0036/former2/files/3948003/former2.com-2576028683490.log)

**Comment by MorganCloud** [comment]
[former2.com-1576028259710.log](https://github.com/iann0036/former2/files/3947968/former2.com-1576028259710.log)

**Comment by iann0036** [comment]
Hi Andrew,

Thanks for reaching out!

I've made a couple of changes that may help us at least get visibility into these errors. Does the credentials you are using have the `ReadOnlyAccess` policy attached or something custom? The SQS: GetQueueAttributes permission is required for this section however that _should_ be included in the ReadOnlyAccess policy.

As for IAM, are you saying the IAM listings used to work and now don't? If so, the only thing I can think of . that's changed recently is the Access Analyzer resource type that has been added (so I've also safely continued if this gets an issue).

I notice your screenshot shows Glue also, which others have reported but I struggle to get to the bottom of. Are you having issues here also?

Regards,
Ian.

**Comment by MorganCloud** [comment]
Thank you for the quick response. For sqs policy it is below:
`{
"Version": "2012-10-17",
"Statement": [
{
"Action": [
"sqs:GetQueueAttributes",
"sqs:GetQueueUrl",
"sqs:ListDeadLetterSourceQueues",
"sqs:ListQueues"
],
"Effect": "Allow",
"Resource": "*"
}
]
}arn:aws:iam::aws:policy/AmazonSQSReadOnlyAccess`

For IAM, yes, it was working last week and for some reason stopped, again , I can see all the resources, but cannot access them .Error I get now is User: arn:aws:iam:::user/a is not authorized to perform: access-analyzer:ListAnalyzers on resource: arn:aws:access-analyzer:us-east-1:

**Comment by iann0036** [comment]
Cheers for that Andrew!

I'm not sure why the policy you've posted would block the call provided, it looks good to me. I'll do some testing of my own.

I did find that the IAM issue is due to the fact Access Analyzer is not part of the ReadOnlyAccess policy yet, so I'll be waiting on AWS to update the policy. In the meantime, including IAMAccessAnalyzerReadOnlyAccess policy on your entity would work.

For both of those areas, I've pushed a change to safely ignore those errors and continue. It will still let you know with an error bubble, but shouldn't block you from generating resources. Please let me know if this is not the case and if it still doesn't unblock the UI.

**Comment by MorganCloud** [comment]
Hello Ian,

Thank you so very much for the quick responses and all the assistance, it is all working now.

Thank you again.
